portfolio/src/main/webapp/js/script.js

```diff
@@ -76,8 +76,19 @@ async function getNumComments(numComments) {
   const messages = await response.json();
   const messageContainer = document.getElementById('message-container');
   messageContainer.innerHTML = '';
-  for (const message of messages) { 
-    messageContainer.appendChild(createListElement(message));
+  document.getElementById('login').setAttribute('href', messages.url);
+  if (JSON.parse(messages.loggedin)) { 
+    document.getElementById('login').innerHTML = 'Logout';
+    for (const message of JSON.parse(messages.comments)) { 
+      messageContainer.appendChild(createListElement(message, /*deleteable=*/messages.user == message.userEmail));
```

Comment:
let's be consistent with spelling. I think `deleteable` -> `deletable` is what you want here, although both look a bit funny after looking at them for a while.